packages/web/src/js/FileUploader.ts

```diff
@@ -86,8 +88,15 @@ class FileUploader extends BaseComponent {
     }
   }
 
-  static getUpdatedFileName(name: string): string {
-    return `file__${name.replace(/\./g, '_').replace(/\s/g, '_')}`;
+  static getUid(): string {
+    return Math.random().toString(36).slice(-6);
+  }
+
+  getUpdatedFileName(name: string): string {
+    return `${this.instanceUid}_file__${name
+      .replace(/\./g, '_')
```

Comment:
Couldn't we unify these regexes into one?

Reply:
Sure.

Reply:
Fixed.